.github/workflows/ci.yml

```diff
@@ -50,7 +49,7 @@ jobs:
         run: "composer audit"
 
       - name: "Run coding style"
-        if: ${{ matrix.dependencies == 'locked' && matrix.php-version == '8.0' }}
+        if: ${{ matrix.dependencies == 'locked' }}
```

Comment:
Change to 8.1 instead of removing I guess?

Reply:
Yeah sure, it's only there as an optimization really at this point - it was previously there as php-cs-fixer didn't run on 8.1 / 8.2.

Reply:
```suggestion
        if: ${{ matrix.dependencies == 'locked' && matrix.php-version == '8.1' }}
```